package.json

```diff
@@ -81,11 +77,11 @@
           ]
         }
       ],
-      "artifactName": "${name}_macos.${ext}"
+      "artifactName": "${name}_macos_installer.${ext}"
     }
   },
   "keywords": [],
-  "license": "MIT",
+  "license": "GNU",
```

Comment:
Should we add the GNU version ? What's the general practise here ?

Reply:
The current license in this repo was GNU so I wrote that.  General practice is whatever license is in repo that name should come here.

Reply:
We use gnu v3 or later, my question was if people add that version here in general.